integration/containerd/cri/integration-tests.sh

```diff
@@ -190,7 +190,7 @@ check_daemon_setup() {
 
 	if [ "${BAREMETAL}" == true ]; then
 		#restart docker service as TestImageLoad depends on it
-		restart_docker
+		sudo -E restart_docker
 	fi
```

Comment:
Why do we need "sudo" here? And it lead to the failure in arm ci.
`10:49:34 sudo: restart_docker: command not found`